extensions/smallrye-reactive-messaging-rabbitmq/deployment/src/main/java/io/quarkus/smallrye/reactivemessaging/rabbitmq/deployment/RabbitMQBuildTimeConfig.java

```diff
@@ -3,27 +3,26 @@
 import java.util.Optional;
 
 import io.quarkus.runtime.annotations.ConfigDocSection;
-import io.quarkus.runtime.annotations.ConfigItem;
 import io.quarkus.runtime.annotations.ConfigPhase;
 import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
 
-@ConfigRoot(name = "rabbitmq", phase = ConfigPhase.BUILD_TIME)
-public class RabbitMQBuildTimeConfig {
+@ConfigRoot(phase = ConfigPhase.BUILD_TIME)
+@ConfigMapping(prefix = "quarkus.rabbitmq")
```

Comment:
@ozangunalp @cescoffier the config for the Messaging extensions is rather inconsistent. Some of them have `quarkus.messaging.whatever` and some of them `quarkus.whatever`. I would have expected the consistent `quarkus.messaging` prefix.

It's out of the scope of this PR though and I will let you handle it once this is in if you want to adjust.